src/components/Trip.jsx

```diff
@@ -74,49 +160,94 @@ const Activities = props => {
             options={activities}
             onChange={(e, data) => setActivityType(data.value)}
           />
-          <h3>Number of times:</h3>
+          <h4>Number of times:</h4>
           <Dropdown
             placeholder="How many times?"
             fluid
             selection
             options={number}
             onChange={(e, data) => setActTimes(data.value)}
           />
-          <Button onClick={onFindActivities}>Find activities</Button>
+          <Button onClick={findActivities}>Find activities</Button>
           {activitiesMessage}
         </Grid.Column>
         <Grid.Column width={8}>
           <h2>Details of trip:</h2>
           <h4>Hotel budget</h4>
           <input
             type="range"
-            name="budget"
+            name="hotel"
             min="1"
             max="5"
             id="slider"
             onChange={sliderChoice}
           ></input>
-          <div className="range-values">
-            <div className="dollar" id="d1">
+          <div className="range-values-hotel">
+            <div className="scale" id="hotel1">
               <h3>✩</h3>
             </div>
-            <div className="dollar" id="d2">
+            <div className="scale" id="hotel2">
               <h3>✩✩</h3>
             </div>
-            <div className="dollar" id="d3">
+            <div className="scale" id="hotel3">
               <h3>✩✩✩</h3>
             </div>
-            <div className="dollar" id="d4">
+            <div className="scale" id="hotel4">
               <h3>✩✩✩✩</h3>
             </div>
-            <div className="dollar" id="d5">
+            <div className="scale" id="hotel5">
               <h3>✩✩✩✩✩</h3>
             </div>
           </div>
-          <Button onClick={findHotels}>Find Hotels</Button>
-          {hotelsMessage}
+          <Button onClick={findHotels}>Check for hotels</Button>
+          <p>{hotelsMessage}</p>
+          <div className="food-choice">
+            <h4>What food do you prefer? </h4>
+            <Dropdown
+              placeholder="Mediterreanean"
```

Comment:
Why not Everything as placeholder (and default value)?

Reply:
Makes sense, I'll use null as default value tho as it will be entered as a keyword to google api.